### PR TITLE
add initial print template

### LINKED ISSUE #10: As an application submitter, I would like be able to print my permit once it is approved.

### PR DESCRIPTION
**Pull request by lauraGgit**
Close #10.

### CODE REVIEW

File: forestserviceprototype/specialuseform/templates/specialuseform/print_permit.html [hunk 1]
```diff
@@ -0,0 +1,57 @@
+{% load static from staticfiles %}
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+      <meta charset="utf-8">
+      <meta http-equiv="X-UA-Compatible" content="IE=edge">
+      <title>{% block title %}Forest Service Prototype{% endblock %}</title>
+      <link rel="stylesheet" href="{% static "additional.css" %}">
+  </head>
+  <body>
```
Comment (colinpmacarthur) [inline forestserviceprototype/specialuseform/templates/specialuseform/print_permit.html:10]: Could we add some kind of instructions at the top, like "**Print this page and take it with you during your activity**?"

**Review by colinpmacarthur — COMMENTED** [review]
Also wasn't able to view this locally -- maybe we should set up a time to screen share?

**Comment by stvnrlly** [comment]
I've rebased this pull request against `develop` to try to remove duplicate commits, and am about to force-push it. If you've been working on this branch locally, you'll need to overwrite the local version of this branch with the new one.

**Comment by stvnrlly** [comment]
This is mergeable, though the actual content of the printed permit has a bunch of placeholder text and empty tags.

If that's good for the prototype, just give me the 👍 and I'll click that button.

**Comment by colinpmacarthur** [comment]
I think that's OK for now @stvnrlly

**Comment by stvnrlly** [comment]
🚀

**Pull request merged**